content/_data/changelogs/weekly.yml

```diff
@@ -429,6 +429,35 @@
       references:
         - url: https://jenkins.io/security/advisory/2017-04-26/
           title: security advisory
+- version: "2.58"
+  date: 2017-04-30
+  changes:
+    - type: rfe
+      message: Use build display names in RSS feed titles.
+      pull: 2845
+    - type: rfe
+      message: >
+        Update the Trilead SSH library to get support of new Mac, Key, and Key Exchange Algorithms.
+      references:
+        - issue: 33021
+        - issue: 26379
+        - issue: 31549
+        # pull: 2848
```

Comment:
The changelog script I wrote only links to `issue` (if that's defined), rather than linking to not `issue` and `pull`.

In the case of 'references' (to be used for more flexible references, and multiple issue links), this is not the case, and it links to every element that's defined. So I add references to the PRs for documentation purposes, but comment them out so they don't show up while the behavior is as currently implemented to match the "single issue reference" presentation.